data/presets/fields/flats.json

```diff
@@ -0,0 +1,7 @@
+{
+    "key": "flats",
```

Comment:
The key for this is actually `building:flats`. Could you change this and move the file to `data/presets/fields/building/flats.json`?

Reply:
Hi  @quincylvania 
Thank you for the feedback. I have made the changes as requested.

Reply:
Looks like you added the new file, but you'll also need to delete this file :)